mapflow/helpers.py

```diff
@@ -105,4 +106,19 @@ def raster_layer_is_allowed(layer: QgsRasterLayer):
         and filepath.suffix.lower() in ('.tif', '.tiff') \
         and filepath.exists() \
         and filepath.stat().st_size < config.MAX_FILE_SIZE_BYTES
-    return res
\ No newline at end of file
+    return res
+
+
+def check_aoi(aoi: Union[QgsGeometry, None]) -> bool:
+    """Check if aoi is within the limits of [[-360:360] [-90:90]]"""
+    if not aoi:
+        return False
+    coordinates = json.loads(aoi.asJson()).get('coordinates')[0]
```

Comment:
What if it is MultiPolygon, not Polygon? I think that indexing will fail then.

1. Most probably we should process separately cases polygon/multipolygon/other(fail with message)
2. Can we access the coordinates of QgsGeometry witthout transforming to Json and traversing the coordinates? It is bettter to take BoundingBox of QGSGeometry and cheeck its 4 bounds than to walk through json.